test/controllers/contactNumbers/ConfirmLandlineNumberControllerSpec.scala

```diff
@@ -24,35 +24,34 @@ import org.mockito.Mockito.reset
 import play.api.http.Status
 import play.api.http.Status.{CONFLICT, INTERNAL_SERVER_ERROR}
 import play.api.test.Helpers._
-import views.html.contactNumbers.ConfirmContactNumbersView
+import views.html.contactNumbers.ConfirmLandlineNumberView
 
 import scala.concurrent.Future
 import assets.BaseTestConstants._
-import audit.models.ChangedPhoneNumbersAuditModel
 import common.SessionKeys
 
-class ConfirmContactNumbersControllerSpec extends ControllerBaseSpec  {
+class ConfirmLandlineNumberControllerSpec extends ControllerBaseSpec  {
 
-  val controller = new ConfirmContactNumbersController(
+  val controller = new ConfirmLandlineNumberController(
     mockErrorHandler,
     mockVatSubscriptionService,
-    injector.instanceOf[ConfirmContactNumbersView],
+    injector.instanceOf[ConfirmLandlineNumberView],
```

Comment:
TestUtil now extends Injecting. So any injects can now be inject[ instead of injector.instanceOf[